queries/useV3ConversationListQuery.ts

```diff
@@ -91,6 +94,17 @@ const v3ConversationListQueryConfig = (
   enabled: !!account,
 });
 
+export const createV3ConversationListQueryObserver = (
+  account: string,
+  context: string,
+  includeSync: boolean = true
+) => {
```

Comment:
when we have 3+ args I think it's better to do object args. I do even when I have 2